tests/integrations/server/kv_service.rs

```diff
@@ -1562,3 +1564,24 @@ fn test_forwarding_reconnect() {
     let resp = client.raw_get_opt(&req, call_opt.clone()).unwrap();
     assert!(!resp.get_region_error().has_store_not_match(), "{:?}", resp);
 }
+
+#[test]
+fn test_health_check() {
```

Comment:
should we cover `watch` too?

Reply:
This only means to check if changes are made by TiKV. Whether `watch` works should guaranteed by grpcio itself.